ddp-workspace/projects/ddp-dsm-ui/src/app/filter-column/filter-column.component.ts

```diff
@@ -87,4 +87,8 @@ export class FilterColumnComponent implements OnInit {
     this.dataFilter.notEmpty = false;
     this.dataFilter.empty = false;
   }
+
+  hideCheckbox(dataFilter: Filter): boolean {
```

Comment:
Is there anything we can do on the backend to give the frontend more information about not showing the "No" option more cleanly?

Reply:
@pegahtah Can you please help answer this question? I'm still learning, I don't know how to do that on the backend.

Reply:
@aweng98 I will check the structure and let you know

Reply:
@aweng98 @zyme I checked backend and db, and we do not specify what options each checkbox gets. The checbox field by default gets `YES` and `NO` and it is set on frontend. So I think this is the best approach for removing `No`